inputFiles/compositionalMultiphaseWell/simpleCo2InjTutorial_base.xml

```diff
@@ -58,7 +58,7 @@
   <ElementRegions>
     <CellElementRegion
       name="reservoir"
-      cellBlocks="{ cellBlock }"
+      cellBlocks="{ all }"
```

Comment:
I do not quite get the interest of having a dedicated keyword when we can have `cellBlocksMatch="{ * }"` with the same result and less corner casees?

Reply:
I forgot to change this one too!

I look in all examples, I'm wondering if I should change all exemples that want to select all mesh in a region to `cellBlocksMatch="{ * }"`. What do you think?

Reply:
Similar question, should I change all examples that want to select all cellblocks with a given `regionAttribute` to `cellBlockAttributeValues={ x }`?
At `inputFiles/poromechanics/nonlinearAcceleration/smallEggModel/smallEggModel.xml:157`, `162`, `167` and `172`, we would just have `cellBlockAttributeValues` to 1, 3, 2 and 0 respectively.

Reply:
@castelletto1 I think you should have a look at this PR (or at least at what it implies) since it can change a lot our `xml` input files (for the better :wink:).
Could you answer @MelReyCG 's question (above in this thread) about spreading the changes to our integrated tests? I think that should be a good idea, but I'm not 100% sure.

@MelReyCG could you update the PR description so we can use it for the announce of the change (I think the people have the right to know!).
Could you also tell me where the `all` keyword comes from? Did you add it yourself? I did not find it in your PR. 🤷

Reply:
About the `all`, I just forgot to change the `cellBlocks="{ all }"` to `cellBlocksMatch="{ * }"` in that example.

Reply:
So I've discussed with @castelletto1 and yes, please modify the integrated tests accordingly.

Reply:
@TotoGaz I did changed the doc and all tests in the GEOS repo accordingly 👍

Reply:
I think we need a `require baseline` label here but I am not sure.